ocaml/xapi/cli_operations.ml

```diff
@@ -1495,6 +1495,42 @@ let sr_probe printer rpc session_id params =
   with _ ->
     printer (Cli_printer.PList [txt])
 
+let sr_probe_ext printer rpc session_id params =
+  let host = parse_host_uuid rpc session_id params in
+  let _type = List.assoc "type" params in
+  let device_config = parse_device_config params in
+  let sm_config = read_map_params "sm-config" params in
+  let results = Client.SR.probe_ext ~rpc ~session_id ~host ~device_config ~_type ~sm_config in
+  let srs, configs =
+    List.fold_left (fun (srs, configs) x ->
+        match x.API.probe_result_sr with
+        | Some sr -> (sr::srs, configs)
+        | None -> (srs, x.API.probe_result_configuration :: configs)
+      )
+      ([], [])
+      results
+  in
+  let print_sr x =
+    let health_to_string = function `healthy -> "healthy" | `recovering -> "recovering" in
+    (match x.API.sr_stat_uuid with
+     | Some uuid -> [ "uuid", uuid ]
+     | None -> []) @
+    [ "name-label", x.sr_stat_name_label
+    ; "name-description", x.sr_stat_name_description
+    ; "total-space", Int64.to_string x.sr_stat_total_space
+    ; "free-space", Int64.to_string x.sr_stat_free_space
+    ; "clustered", string_of_bool x.sr_stat_clustered
+    ; "health", x.API.sr_stat_health |> health_to_string
```

Comment:
Using `|>` here is a slight inconsistency as it could have been used above as well for the other values.